spel/minimal-linux.json

```diff
@@ -209,6 +209,26 @@
             "temporary_security_group_source_cidrs": "{{ user `security_group_cidrs` }}",
             "type": "amazon-ebs",
             "user_data_file": "{{ template_dir }}/userdata/tmpfsroot-el8.cloud"
+        },
+        {
+            "flavor": "{{ user `openstack_flavor_name` }}",
+            "floating_ip_network": "{{ user `openstack_floating_ip_network_name`}}",
+            "image_name": "{{ user `spel_identifier` }}-{{ build_name }}-{{ user `spel_version` }}.x86_64-gp2",
```

Comment:
Does OpenStack have `gp2` storage-devices? If not, you might want to omit the `-gp2` from the name. Or, if OpenStack offers different storage-options for the instances that will be launched from the template, I'd change `-gp2` to whatever storage-type the image is intended to provide default support to?

Reply:
I haven't gotten OpenStack Cinder set up yet, but I'm pretty sure it supports breaking out backing storage by SSD (like AWS "GP2") and HDD (like AWS "Magnetic").  We're trying to keep the naming conventions for resources in our OpenStack environment similar to what can be found in AWS to ease tenant transition back and forth, which is why I use the same naming convention you folks are using (e.g. https://github.com/plus3it/spel/blob/33dad869ccc9dbec1a9da7003f806923ec2d5242/spel/minimal-linux.json#L33).

Reply:
Alright. Cool. Thank you for the explanation.

Last I'd looked at OpenStack (five years ago, at this point?), its backing-store was something akin to AWS's instance-store AMI-types. This project didn't get off the ground in time to need to make builders for instance-store. However, it was still common to see AMIs for instance-store and AMIs for EBS. At that point in time, most of what we observed were being labeled `gp2` – presumably because AWS was already making `gp2` the new basic-storage tier vice the `st1` that overlapped with instance-store. 

Could you add a note to your documentation-content to reflect the "why" of the naming-choice?

Reply:
Interesting this is coming up, since we'll shortly be testing changing the volume type to `gp3`. Most likely, since we need to support regions that may not have `gp3` volumes yet, we'll expose a new var for the aws volume type. I'd recommend using a variable here for the openstack volume type.

Reply:
I went ahead and removed the -gp2 suffix altogether; I hope that's acceptable.  I'm not entirely sure Nova-backed (think AWS "instance-backed") storage even supports specifying a backing storage type (I think you just get whatever Nova on the host you land on is configured to use).  If I find out I'm wrong, and we end up storing images that default to SSD vs. HDD, I'll open a new PR to add this variable in.  (https://github.com/plus3it/spel/pull/439/commits/53100ec0c7bacae3970a69b1aaf2357007d36ac9)